tests/test_spotify_service.py

```diff
@@ -0,0 +1,63 @@
+import os
+import pytest 
+from dotenv import load_dotenv
+
+from app.spotify_service import SpotifyService
+service = SpotifyService()
```

Comment:
I think if we move this line into each skipped test, then we shouldn't see this code getting invoked on CI.

The downside to that is multiple auth requests, one per service initialization. So instead we could try using a fixture instead. Will share an example here in a sec

Reply:
https://github.com/prof-rossetti/intro-to-python/blob/main/notes/python/packages/pytest.md#fixtures

try something like this:

```py
@pytest.fixture(scope="module")
def spotify_service():
    return Spotify Service()


```

Then we just need to pass the `spotify_service` car as a parameter to the test functions that need it.

Reply:
Thank you! This is working locally, I'll see if it works in CI